package.json

```diff
@@ -58,7 +56,7 @@
     "css": "^2.2.4"
   },
   "peerDependencies": {
-    "styled-components": "^2.0.0 || ^3.0.2"
+    "styled-components": "^2.0.0 || ^3.0.2 || >= 4.x"
```

Comment:
Using `babel-preset-env` is OK, but this library [is not ready yet](https://github.com/styled-components/jest-styled-components/issues/183#issuecomment-419975878) for supporting v4.

Reply:
I see. I'm not using enzyme, it is working with `react-test-renderer`. But I understand why you want to wait on adding that as a peerDependency.

Reply:
Correct.
I was thinking about deprecate Enzyme, but I believe a lot of people find it useful.